pymetis/__init__.py

```diff
@@ -243,6 +243,12 @@ def part_mesh(n_parts, connectivity, options=None):
     METIS runtime options can be specified by supplying an :class:`Options`
     object in the input.
 
+    ``tpwgts`` is a list of size ``n_parts`` that specifies the desired weight 
+    for each partition.
+
+    ``gtype``, as either metis.GType.NODAL (0) or metis.GType.DUAL (1), 
+    specifies the partition is base on a nodal or dual graph of the mesh
```

Comment:
Does this duplicate the nodal partitioning function that was recently added?

Reply:
I did not copy & paste. I change the function prototype of the original `part_mesh` by adding two arguments with default values. The output of the new `part_mesh` should be consistent with the old one if users do not touch the new arguments.